tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -82,9 +82,13 @@ use fmt::{Debug, Display};
 ///             write!(writer, "{}", span.name())?;
 ///
 ///             let ext = span.extensions();
+///
+///             // `FormattedFields` is a a formatted representation of the span's
+///             // fields which are stored in its extensions
```

Comment:
I expeanded on this a little and fixed a grammatical issue, what do you think?
```suggestion
///             // fields, which is stored in its extensions by the `fmt` layer's
///             // `new_span` method. The fields will have been formatted
///             // by the same field formatter that's provided to the event 
///             // formatter in the `FmtContext`.
```

Reply:
Sounds good. English not be my best :P